api/v1beta2/foundationdb_env_variables.go

```diff
@@ -21,6 +21,75 @@
 package v1beta2
 
 const (
-	// EnvNamePublicIP defines the FDB_PUBLIC_IP environment variable name.
+	// EnvNameDNSName specifies the DNS locality (identifies the pod when using DNS in lieu of IP)
+	EnvNameDNSName = "FDB_DNS_NAME"
+
+	// EnvNameMachineID specifies the Machine ID locality. Also defines fault domain along with EnvNameZoneID
+	EnvNameMachineID = "FDB_MACHINE_ID"
+
+	// EnvNameZoneID specifies Zone ID locality. Also defines fault domain along with EnvNameMachineID.
```

Comment:
```suggestion
	// EnvNameZoneID specifies Zone ID locality.
```

There can be configurations where a machine is used as fault domain, in most cases a user will define a fault domain that contains more than a single machine (rack, AZ, data hall, etc.).